src/test/lit-element_test.ts

```diff
@@ -374,4 +374,165 @@ describe('grimlock', () => {
       );
     });
   });
+
+  it('converts mwc-checkbox', () => {
```

Comment:
is this working?

Reply:
Nope, we're missing the boilerplate and property types. I've opened an issue for property types, PR incoming.

I can make this spec an `xit` so it doesn't run for now.